docs/topics/forms/modelforms.txt

```diff
@@ -1230,10 +1230,52 @@ of a model. Here's how you can do that::
 
 Notice how we pass ``instance`` in both the ``POST`` and ``GET`` cases.
 
+Inline formsets and ``related_name``
+------------------------------------
+
+If the foreign key on your child model has a ``related_name`` to the parent,
+the formset uses this name as its ``prefix``.
+
+For example, consider the following models and formset::
+
+    from django.db import models
+    from django.forms import inlineformset_factory
+
+    class Author(models.Model):
+        name = models.CharField(max_length=100)
+
+    class Book(models.Model):
+        author = models.ForeignKey(Author, on_delete=models.CASCADE, related_name='books')
+        title = models.CharField(max_length=100)
+
+    BookFormSet = inlineformset_factory(Author, Book, fields=('title',), extra=1)
+
+``BookFormSet`` above will have its ``prefix`` attribute set to 'books'::
+
+    >>> formset = BookFormSet()
```

Comment:
I'm not sure that this example is needed. The sentence seems to suffice as I think anyone reading at this point knows what an attribute access looks like. :-)